evergreen/pyproject.toml

```diff
@@ -23,6 +23,10 @@ psycopg = { extras = ["binary"], version = "^3.2.3" }
 
 Pillow = "^9.2.0"
 django-bootstrap = "^0.2.4"
+flake8 = "^7.1.1"
+autopep8 = "^2.3.1"
+isort = "^5.13.2"
+mypy = "^1.13.0"
```

Comment:
Does Poetry happen to support dev-envorinment-only packages like npm's `--save-dev`? Obviously we don't need linting/formatting in prod, could save some time on deployment waiting for the Docker container to come online

Reply:
This is technically true but I'd rather not deal with it right now.